### ISSUE TITLE
Testing access to someone else's page.

### ISSUE THREAD

**Issue by hail2skins** [ksimmons/policy-assertions#4]
This is a really great gem I stumbled across today while reading about how to test Pundit with Minitest as I'm new to Pundit and new-ish to Minitest.

As I'm reading your instructions and testing things out, one thing jumped out at me.  

In a permissions model where a particular show page is accessible to one user, but not another based on whatever criteria, it does not appear you can specifically test that using this gem.   Would it be a good idea to have a test like:

```Ruby
def test_show(user(:first_user))
  assert_permit user(:first_user), User
  refute_permit nil, User
  refute_permit user(:other_user), User
end
```

Basically so I'm testing a specific "show" page rather than any show page?

**Comment by ksimmons** [comment]
You have it correct but you don't need the parameter passed into test_show. Here's an example of a project I use this gem on. You can see that I'm testing multiple users with instances of the comment class. Does this help?

```ruby 
def test_show
  assert_permit users(:admin), comments(:regular_user_comment)
  assert_permit users(:moderator), comments(:regular_user_comment)
  assert_permit users(:content_editor), comments(:regular_user_comment)
  assert_permit users(:regular_user), comments(:regular_user_comment)
  assert_permit users(:no_projects_user), comments(:regular_user_comment)
  assert_permit nil, comments(:regular_user_comment)
end
```

You can see more examples here: https://github.com/ksimmons/codedorm

**Comment by hail2skins** [comment]
That's excellent.   Thank you :).
